library/pom.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.homeofthewizard</groupId>
+    <artifactId>spring-bridge-maven-plugin</artifactId>
```

Comment:
can we rename it to smth else ?
I have a plugin called like that exactly.
https://github.com/HomeOfTheWizard/spring-bridge-maven-plugin

Reply:
```suggestion
    <artifactId>spring-bridge-maven-library</artifactId>
```